ChineseFilter/sections/specific.txt

```diff
@@ -23,6 +23,7 @@ $script,xmlhttprequest,third-party,domain=00ksw.com|01zww.com|0biquge.com|114zw.
 !
 !
 !
+mgnacg.com##a[href*="//fanqiev2.work/auth/register"]
```

Comment:
Why `*=`? I see `href="https://fanqiev2.work/auth/register?code=14wq"` only.
Also can be hidden by `.qiafan`

Reply:
The reporter's comment (Translated by Google Translator):
> Open the browser and open the AdGuard plug-in. When entering the website for the first time, the screenshot of the advertisement is not blocked. It can be blocked after manual identification. Even if you log out of the website and enter again, the blocking is still effective. However, the problem reappears when you re-enter the website after a period of time after closing the browser.